coverage/artifacts/local_test.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package artifacts
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"testing"
+
+	"k8s.io/test-infra/coverage/logUtil"
+	"k8s.io/test-infra/coverage/test"
+	"os"
+)
+
+const (
+	covTargetRootRel = "testTarget"
+)
+
+// generates coverage profile by running go test on target package
+func TestProfiling(t *testing.T) {
+	if os.Getenv("GOPATH") != "" {
+		logUtil.LogFatalf = log.Fatalf
+
+		arts := localArtsForTest("TestProfiling")
+		arts.ProduceProfileFile(fmt.Sprintf("../%s/subPkg1/ "+
+			"../%s/subPkg2/", covTargetRootRel, covTargetRootRel))
+
+		t.Logf("Verifying profile file...\n")
```

Comment:
I would suggest not using linefeed in log messages whenever possible. Usually they're unnecessary (e.g., EOL), produce weird blank lines or mess up with a single line copy & paste.

Reply:
removed all